### DISCUSSION TITLE
What is mod_ratio and I can get m6a site without DRACH motif?

### DISCUSSION THREAD

**Discussion by thadasps** [GoekeLab/m6anet#71] [Q&A]
Hi, Thank you for develop this tools. It help me to do this project very well.

So I can run m6anet with sample from Adenovirus type5 and I get the output around 33 site but for all site that occure only in DRACH motif

my question is
1. Can I use m6anet to get m6a site for other k-mer not only DRACH motif (Such as '-A----')
2. What is mean and How I interprete the data in column mod_ratio?

Sorry for my poor english.

**Comment by chrishendra93** [comment]
hi @thadasps 

1. Currently m6anet only supports DRACH motif kmers. If you want to extract features from other motifs you need to modify the kmers in the constant.py file
2. The mod ratio column is the percentage of modified reads for every site detected by m6anet

Thanks

Christopher Hendra
